parsepatch/patch.py

```diff
@@ -286,6 +297,34 @@ def parse_hunks(self, line):
                 break
             self.parse_hunk(line)
 
+    def parse_files_hunks(self, line):
+        (src_start, src_end), (dest_start, dest_end) = self.parse_numbers(line)
+        diff = []
+
+        for h_line in self.get_lines:
+            if h_line.startswith('diff --git a/') or h_line.startswith('diff -r '):
```

Comment:
Is there a way to avoid code duplication ?
I mean we've common code here and in get_hunks(self), so it'd be nice to avoid it.

Reply:
I have now addressed the code duplication by merging get_changes(self) and get_hunks(self).